wirecard/oxid-ee
# WDPD-218: Improve code style checks

### This PR

* Splits up PHPCS rules into two standards; one for base rules (**WirecardBase**) and one for OXID-related rules (**WirecardOxid**)
* Resolves an issue with the **UnusedImportSniff** that caused PHP warnings to be thrown
* Slightly adapts the folder structure to put all code related to code analysis into one place
* Adapts the following sniffs:
  * **ImportSniff**: `use` statements are now forced to be ordered alphabetically (this was already the case for most of the codebase)
  * **ClassLinesOfCodeSniff** / **FunctionLinesOfCodeSniff**: Whitespace and comment tokens are now no longer considered when calculating the total lines of code
  * **DocBlockSniff**: Class properties are now also required to have a DocBlock

### Notes

* `@since` tags are removed from all sniffs now because they should be portable across different plugins

### How to Test

* Run `rake cs_check` and/or `rake cs_fix` and confirm that no new errors are displayed and that there are no PHP notices/warnings

### Jira Links

* WDPD-218: https://jira.parkside.at/browse/WDPD-218
